### DISCUSSION TITLE
Prefix an sort_title to a created dynamic collection possible?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#1048] [Q&A] [answered]
Hello,

I'd like to ask, if it is possible in any way to have PMM prefix sort titles to collections created in Trakt or TMDb ?

I have configured dynamic collections here:
```
dynamic_collections:
   Trakt Liked Lists:
     collection_order: custom
     type: trakt_liked_lists
     data: me
     remove_suffix: Collection, Collections, Filmreihe, Filmreihen
     remove_prefix: The, Die
  TMDb Collections:
    type: tmdb_collection
    remove_suffix: Collection, Collections, Filmreihe, Filmreihen
    remove_prefix: The, Die
```

And like all the created collections to have an sort_title like, say `+++_01_2_<COLLECTION NAME FROM TMDB OR TRATK>` instead of just `<COLLECTION NAME FROM TMDB OR TRATK>`

How would I do this? 
I can't find a wiki page adding a prefix in dynamic collections, only removing.

Thank you very much.

**Comment by meisnate12** [comment, marked as answer]
thats because you cant from the dynamic_collections call you have to edit the sort_title using a template

**Reply by gitttt54** [reply]
Thank you very much!


I have -with all your help- created a template and it seem to work good!

```
templates:
  my-template4TraktLikedLists:
    trakt_list_details: <<value>>
    sort_title: +++01_2_<<collection_name>>

dynamic_collections:
  Trakt Liked Lists:
    template: my-template4TraktLikedLists
    type: trakt_liked_lists
    data: me
  
```
